packages/session-replay-browser/src/session-replay.ts

```diff
@@ -460,7 +464,9 @@ export class SessionReplay implements AmplitudeSessionReplay {
 
   stopRecordingEvents = () => {
     try {
-      this.loggerProvider.log('Session Replay capture stopping.');
+      if (this.recordCancelCallback) {
```

Comment:
I may make this change on v1.x separately as well, this log is confusing/misleading when this method is called but we're not actually recording